l5kit/l5kit/rasterization/sat_box_rasterizer.py

```diff
@@ -34,9 +36,11 @@ def __init__(
         self.world_to_aerial = world_to_aerial
         self.interpolation = interpolation
 
-        self.box_rast = BoxRasterizer(raster_size, pixel_size, ego_center, filter_agents_threshold, history_num_frames)
+        self.box_rast = BoxRasterizer(
```

Comment:
don't you already have some of these information in the `render_context`?

Reply:
you mean `raster_size`?

Reply:
isn't:
 `raster_size_px` == `raster_size`
`pixel_size_m` == `pixel_size`
`center_in_raster_ratio` == `ego_center`
?